index.js

```diff
@@ -4,22 +4,87 @@
  * @param {Array} vars - array of usable variables entered by user
  * @returns converted fhirpath expression
  */
-export function fhirconvert(str, vars) {
+ export function fhirconvert(str, vars) {
   if (validate(str, vars)) {
-    return varfind(convert(str), vars);
+    return varfind(convert(str.trim()), vars);
   } else {
     return null;
   }
 }
 
 /**
  * Verifies normal syntax by confirming var names, function names,
- * and number of parenthesis.
+ * syntax, and number of parenthesis.
  * @param {string} str - inputted normal syntax expression
  * @param {Array} vars - array of usable variables
  * @returns boolean, valid or invalidh
  */
 export function validate(str, vars) {
+/*
+  let ops = [
```

Comment:
It would be better to remove these entirely so we can see the addition and comment there in the next PR.  Otherwise, we would have to review all these lines here.
It if it helps, you can remove them in a single commit (without other changes) and then revert the commit in your next PR.

Reply:
Fixed